### PR TITLE
make hostapd's max_num_sta configurable

### LINKED ISSUE #351: Limit wifi to N clients
Hi,

How to limit the hotspot for only 4 concurrent users ?

### PR DESCRIPTION
**Pull request by glaszig**
resolves #351.

screenshot

### CODE REVIEW

**Comment by billz** [comment]
This is shaping up to be an awesome week for the project. Again, many thanks @glaszig

File: includes/hostapd.php [hunk 1]
```diff
@@ -225,6 +225,13 @@ function DisplayHostAPDConfig()
                     </div>
                   </div>
                 </div>
+                <div class="row">
+                  <div class="form-group col-md-4">
+                    <label for="max_num_sta"><?php echo _("Maximum number of clients") ?></label>
+                    <input type="text" id="max_num_sta" class="form-control" name="max_num_sta" placeholder="2007" value="<?php echo $_POST["max_num_sta"] ?>" aria-describedby="max_num_sta_help">
+                    <span id="max_num_sta_help" class="help-block">Configures the <code>max_num_sta</code> option of hostapd. The default and maximum is 2007. If empty or 0, the default applies.</span>
```
Comment (billz) [inline includes/hostapd.php]: In testing, I noticed the `max_num_sta` value does not persist in the input field. The `$_POST` value here should be replaced with `$arrConfig['max_num_sta']`.

The help text in this block should be output with the gettext() alias `echo _()` to support language translations. I will then work on localizing this message in the project's supported languages.

**Comment by billz** [comment]
@glaszig nice addition. I've noted a couple of small items in the commit review. thanks

**Comment by glaszig** [comment]
> couple of small items

sorry, done.

**Comment by billz** [comment]
super, thank you

**Pull request merged**